docs/root/configuration/best_practices/_include/edge.yaml

```diff
@@ -48,6 +48,7 @@ static_resources:
             tls_certificates:
             - certificate_chain: {filename: "certs/servercert.pem"}
               private_key: {filename: "certs/serverkey.pem"}
+            alpn_protocols: ["h2,http/1.1"]
```

Comment:
this could probably do with some more documenting somewhere, grepping repo, and searching docs doesnt turn up too much (api docs are here https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/tls.proto.html)

other than api refs there is no mention in docs, and no usage in the sandboxes

perhaps beyond the scope of this and not sure if we want to add anything on the edge config page - https://www.envoyproxy.io/docs/envoy/latest/configuration/best_practices/edge.html

Reply:
example configs with tls + HCM at all are
examples/brotli/brotli-envoy.yaml
examples/front-proxy/front-envoy.yaml
examples/tls/envoy-https-https.yaml
examples/tls/envoy-https-http.yaml
examples/tls-sni/envoy.yaml
examples/websocket/envoy-wss.yaml
examples/win32-front-proxy/front-envoy.yaml

I could add to the tls examples?  Or all? But mainly I just want to fix what's broken :-)

Reply:
> I just want to fix what's broken :-)

Yep, cool, follow up added here #19484